### ISSUE TITLE
Feedzy cache is not working for the widget

### ISSUE THREAD

**Issue by vytisbulkevicius** [Codeinwp/feedzy-rss-feeds#677]
### Description:
Client reported that if he sets a specific cache time for a feed in the widget:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/36594177/189891095-b880d8e1-216c-464c-b7da-9dd1ba87a0b0.png">

it's being ignored and they see the data being changed after a few hours.

I tried to replicate this myself and it's not working as expected, I set the cache to 15 days (meaning that even if there are new items in the feed or something is removed Feedzy shouldn't try to fetch new data for 15 days) and after a few hours I checked the page - data was changed, and new items were fetched/removed.

### How to reproduce:
1. Create a new widget, add feed as a source, fetch data and set the cache time to 15 days
2. Save the page/post and leave it like this
3. Make sure something changed in the feed you're using and check the page after a few hours - you'll see new data appearing

### Expected behaviour:
Feedzy should cache the data for the time selected
### Current behaviour:
Cache time doesn't work
### Reference:
Reported here - https://secure.helpscout.net/conversation/1982743742/333665

### Technical info
* WordPress version: 6.0.2
* Plugin version: 4.0.2

**Comment by vytisbulkevicius** [comment]
@girishpanchal30,

I confirmed that I see the same timestamp in the database, which looks to be set correctly when I saved the widget. I checked the timestamp and also see 15 days, however, it doesn't look to be working.

I set the new widget with a feed here - https://feedzy.s.themeisle.com/testpage/
I set it to be refreshed after 15 days but I was constantly checking the page with refreshing it and after about 1+ hour I noticed that items got refreshed.

Previously when reported the issue I used client's feed but now to make things easier I used a test feed from another test instance - https://enchantingcreator.s2-tastewp.com/feed/

So I used this feed, after widget was saved I created another post in that test instance and it appeared on the feed: https://vertis.d.pr/X1cYWZ

After 1+ hour I noticed that this item appeared on my test page also even though it shouldn't appear there for 15 days: https://vertis.d.pr/WIUxwM
